pdf_parsing/test_parser_result.py

```diff
@@ -0,0 +1,300 @@
+import json
+import unittest
```

Comment:
Thanks for adding tests, have you considered using pytest?

Reply:
Rewrote tests to use `pytest` instead of `unittest`.

Reply:
wow awesome, you did not have to ;)
but I hope you felt the joy of using pytest :)